var/spack/repos/builtin/packages/llvm-openmp-ompt/package.py

```diff
@@ -33,26 +33,53 @@ class LlvmOpenmpOmpt(CMakePackage):
 
     homepage = "https://github.com/OpenMPToolsInterface/LLVM-openmp"
 
-    # tr4-stable branch
+    # towards_tr4 branch
+    version('towards_tr4', branch='towards_tr4',
+            git='https://github.com/OpenMPToolsInterface/LLVM-openmp.git')
+
     version('3.9.2b2',
             git='https://github.com/OpenMPToolsInterface/LLVM-openmp.git',
             commit='5cdca5dd3c0c336d42a335ca7cff622e270c9d47')
+
     # align-to-tr-rebased branch
     version('3.9.2b',
             git='https://github.com/OpenMPToolsInterface/LLVM-openmp.git',
             commit='982a08bcf3df9fb5afc04ac3bada47f19cc4e3d3')
 
+    # variant for building llvm-openmp-ompt as a stand alone library
+    variant('standalone', default=False,
+            description="Build llvm openmpi ompt library as a \
+                         stand alone entity.")
+
     depends_on('cmake@2.8:', type='build')
-    depends_on('llvm')
+    depends_on('llvm', when='~standalone')
     depends_on('ninja@1.5:', type='build')
 
     generator = 'Ninja'
 
     def cmake_args(self):
-        return [
-            '-DCMAKE_C_COMPILER=clang',
-            '-DCMAKE_CXX_COMPILER=clang++',
+        cmake_args = [
+            '-DCMAKE_BUILD_TYPE=Release',
```

Comment:
i think it's already in `CMakePackage` package. You can check the build logs to be sure

Reply:
p.s. you can overwrite build variant provided in the parent `CMakePacage` via
```
    variant('build_type', default='Release',
            description='The build type to build',
            values=('Debug', 'Release'))
```

The default is:
```
build_type [RelWithDebInfo]    Debug, Release,         CMake build type
                                   RelWithDebInfo,
                                   MinSizeRel
```

Reply:
@davydden Should I do something now, or fix it next time?

Reply:
@jgalarowicz while you are waiting for someone to merge, just fix it now, it's straightforward.